docs/aws_lambda_support.md

```diff
@@ -0,0 +1,63 @@
+# Support for the AWS Lambda Platform
+
+This document discusses the support offered in this repository to running benchmarks as containerized functions on AWS Lambda. Currently, AWS Lambda support has been implemented for the _map-reduce_ benchmark. Please note that tracing is not supported on AWS Lambda for any benchmark currently.
```

Comment:
Open an issue for adding tracing in future.

Reply:
Done. Issue [#383](https://github.com/vhive-serverless/vSwarm/issues/383).